WalletWasabi.Gui/Controls/WalletExplorer/SendTabViewModel.cs

```diff
@@ -337,15 +337,7 @@ public SendTabViewModel(WalletViewModel walletViewModel, bool isTransactionBuild
 						MainWindowViewModel.Instance.StatusBar.TryRemoveStatus(StatusBarStatus.DequeuingSelectedCoins);
 					}
 
-					BuildTransactionResult result;
-					if (useCustomFee)
-					{
-						result = await Task.Run(() => Global.WalletService.BuildTransaction(Password, new[] { operation }, feeTarget: null, feeRate: Money.Satoshis(feeRate.SatoshiPerByte), allowUnconfirmed: true, allowedInputs: selectedCoinReferences));
-					}
-					else
-					{
-						result = await Task.Run(() => Global.WalletService.BuildTransaction(Password, new[] { operation }, feeTarget: FeeTarget, feeRate: null, allowUnconfirmed: true, allowedInputs: selectedCoinReferences));
-					}
+					BuildTransactionResult result = await Task.Run(() => Global.WalletService.BuildTransaction(Password, new[] { operation }, feeTarget: null, feeRate: Money.Satoshis(feeRate.SatoshiPerByte), allowUnconfirmed: true, allowedInputs: selectedCoinReferences));
```

Comment:
I refactored the code to always set `SatoshiPerByteFeeRate`in every cases. So when building the transaction we dont have to Parse and figure out the UI logic.